website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -95,7 +95,7 @@ The footer for the Single component includes a total count, which is customizabl
 
 ##### Multiple
 
-The footer for the Multiple component includes buttons to modify the options and a count of the number of options selected out of the total.
+The Multiple component footer includes buttons to modify options and a selected count. This default setup is customizable to fit specific needs.
```

Comment:
First,
[super nit] I've brought back some of the original framing for several reasons. 
1. The footer is the main focus of this section, so including it first reinforces the content's purpose. It also follows the sentence structure of the two paragraphs before it: "By default, the footer..." and "The footer for the Single component..."
2. "modify the options" means the actions (buttons) change/adjust (modify) the options within the list, which "modify options" could be interpreted as the ability to change/adjust (modify) settings or the like. It's less clear what the action is taking place on.

Second,
[nit] This could read like a button can somehow be used for the selected count, so flipping the order can avoid confusion. 

Third,
[blocking] This change was meant to include our copy recommendation, but I don't see that present. While the content is _now_ customizable, it wasn't intended to be (as outlined in the [ticket description](https://hashicorp.atlassian.net/browse/HDS-4136)). 

```suggestion
The footer for the Multiple component includes a selected count and buttons to modify the options. While the content can be customized, we recommend using "{xx} selected of {xx} total" to drive consistency across products.
```

The language for the Single component section just above this also needs to be adjusted to include similar language about the recommendation. Please let me know if you want to discuss further.

Reply:
Ah, okay, that makes sense. I thought adding a snippet similar to the content in the single component section would suffice. I'll make those changes!